overlay-app/src/components/Scoreboard.tsx

```diff
@@ -74,41 +77,53 @@ export const Scoreboard: FC<ReturnType<typeof getState>> = (props) => {
       rightWon--;
     }
 
-    leftMarks = <div className="left">{leftRows}</div>;
+    leftMarks = <div className="tally-side left">{leftRows}</div>;
 
-    rightMarks = <div className="right">{rightRows}</div>;
+    rightMarks = <div className="tally-side right">{rightRows}</div>;
   }
 
   let sizes:Array<[number, string]> = [[13, "2.5rem"], [16, "2rem"], [23, "1.5rem"]];
   let leftFontSize, rightFontSize = "1.5rem";
   [leftTeamName, leftFontSize] = scaleText(leftTeamName, sizes);
   [rightTeamName, rightFontSize] = scaleText(rightTeamName, sizes);
+  let hasBothLogos = leftTeamLogo && rightTeamLogo;
 
   return (
-    <div className="scoreboard">
-      <div className="left">
-        <div className="name" style={{fontSize:leftFontSize}}>{leftTeamName}</div>
-        <div className="score">{teams[0].score}</div>
-      </div>
+    <div className="scoreboard-wrapper">
```

Comment:
Moved the tally score pieces into the left/right side to make the sizes of things relative to their parent container and rely less on the css variables, which could be more complicated when tweaking the sizes of things